architecture/wm_azure/001-sequence.puml

```diff
@@ -0,0 +1,42 @@
+' Copyright (C) 2015-2021, Wazuh Inc.
+' Created by Wazuh, Inc. <info@wazuh.com>.
+' This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+@startuml wm_azure
+    main -> wmodules : wm_config()
+    wmodules -> config : ReadConfig()
+    config -> config : read_main_elements()
+    config -> config : Read_WModule()
+    config -> config : wm_azure_read()
+    config -> config : wm_azure_api_read()
+    wm_azure -> wm_azure_setup : wm_azure_setup(config)
+    wm_azure_setup -> wmodules : wm_state_io()
+loop
```

Comment:
Please add the denote that the loop is part of the mainloop thread that comes from the execution of the wazuh module (aka wm_azure_main) anyway, something I see in the whole diagram is that it shows a tree of "how" calls was made, which is not correct.

the important thing to see in the sequence diagram is the "what" does, let's use descriptive text so as not to have to go to see the code to see what it does

Reply:
Got it. I'll change the approach of the diagram and upload it again.